app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -348,6 +349,45 @@ export default class SampleViewV2 extends React.Component {
     return parsedValue;
   };
 
+  parseSpeciesMetricDecimalPlaces = species => {
+    let parsedSpeciesMetrics = Object.assign({}, species);
```

Comment:
Any reason why yo can't edit species directly instead of a new object? would save space...

Reply:
Sorry! This was because I was comparing the original species vs the species with adjusted decimal places and forgot to change back to species directly